### PR TITLE
[ui-library-quickstart-composites] Maintenance tasks

### PR DESCRIPTION
**Pull request by prprabhu-ms**
## Purpose
* Bump to `@azure/communication-react@1.2`
* Generate the `groupId` and `threadId` in the quickstart app to make it more approachable for developers.
* Add outline to composites for better visual separation.

### CODE REVIEW

File: ui-library-quickstart-composites/src/App.tsx [hunk 1]
```diff
@@ -75,8 +95,58 @@ function App(): JSX.Element {
   return <h3>Initializing...</h3>;
 }
 
-const containerStyle = {
-  height: '50%',
+const containerStyle: CSSProperties = {
+  border: 'solid 0.125rem olive',
+  margin: '0.5rem',
+  width: '50vw',
```
Comment (edwardlee-msft) [inline ui-library-quickstart-composites/src/App.tsx:101]: Are we lenient on the use of vw and vh in the quickstart composites? @JamesBurnside

Comment (JamesBurnside) [reply ui-library-quickstart-composites/src/App.tsx:101]: Yeah, as long as its outside the npm package its totally fine

File: ui-library-quickstart-composites/src/App.tsx [hunk 2]
```diff
@@ -1,30 +1,46 @@
...
+ * Entry point of your application.
+ */
+function App(): JSX.Element {
+  // Arguments that would usually be provided by your backend service or
+  // (indirectly) by the user.
+  const { endpointUrl, userId, token, displayName, groupId, threadId } = useAzureCommunicationServiceArgs();
 
-  // We can't even initialize the Chat and Call adapters without a well-formed token.
+  // A well-formed token is required to initialize the chat and calling adapters.
   const credential = useMemo(() => {
```
Comment (edwardlee-msft) [inline ui-library-quickstart-composites/src/App.tsx:44]: Note for other pr reviewers:
credential is created is here. (was collapsed for me).

**Review by edwardlee-msft — APPROVED** [review]

File: ui-library-quickstart-composites/src/App.tsx [hunk 3]
```diff
@@ -1,30 +1,46 @@
...
-  // Provide any valid UUID for `groupId`.
-  const groupId = '<Developer generated GUID>';
-  // Chat Variables
-  const threadId = '<Get thread id from chat service>';
+/**
+ * Authentication information needed for your client application to use
+ * Azure Communication Services.
+ *
+ * For this quickstart, you can obtain these from the Azure portal as described here:
+ * https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/identity/quick-create-identity
```
[the remark this answers is not in the document]
Comment (prprabhu-ms) [reply ui-library-quickstart-composites/src/App.tsx:18]: I couldn't find anything that documents that nicely hidden away endpointUrl value...

**Review by dmceachernmsft — APPROVED** [review]

**Pull request merged**